scaladoc/src/scala/tasty/inspector/Inspector.scala

```diff
@@ -0,0 +1,33 @@
+// Copy of tasty-inspector/src/scala/tasty/inspector/Inspector.scala
+// FIXME remove this copy of the file
```

Comment:
The `scaladoc` project depends on `scala3-tasty-inspector`, but for some reason, some of the use cases do not include this dependency at runtime. It fails with `Caused by: java.lang.ClassNotFoundException: scala.tasty.inspector.Inspector` (see `community-build` and `sbt-test/scripted`). The previous version did not use the proper inspector classes either and therefore did not encounter this issue.

It is fine to have a copy of these files here for now as it is equivalent to the current state. Someone will need to investigate and fix this issue later.